admin_guide/image_policy.adoc

```diff
@@ -0,0 +1,154 @@
+[[admin-guide-image-policy]]
+= Image Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[image-policy-overview]]
+== Overview
+
+You can control which images are allowed to run on your cluster using the ImagePolicy
+admission plugin (currently considered beta).  It allows you to control:
```

Comment:
> Is beta the same thing as "technology preview"?

Similar: https://github.com/openshift/origin/#alpha-and-unsupported-kubernetes-features .  I think this is beta.